livekit-plugins/livekit-plugins-openai/livekit/plugins/openai/realtime/realtime_model.py

```diff
@@ -66,6 +69,10 @@ class RealtimeResponse:
     """usage of the response"""
     done_fut: asyncio.Future[None]
     """future that will be set when the response is completed"""
+    created_timestamp: float
+    """timestamp when the response was created"""
+    first_token_timestamp: float | None = None
+    """timestamp when the first token was received"""
```

Comment:
We should keep those states private

Reply:
Renamed to `_created_timestamp` and `_first_token_timestamp`.